src/Parser/Warrior/Fury/Modules/Features/Abilities.js

```diff
@@ -0,0 +1,163 @@
+      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown: haste => 50, // TODO: Add custom function that depends on CoF (and Odyn's Champion) (RNG)
+      recommendedEfficiency: 0.95,
+    },
+    {
+      spell: SPELLS.ODYNS_FURY,
+      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown: haste => 45,
+      recommendedEfficiency: 0.95,
+    },
+    {
+      spell: SPELLS.AVATAR_TALENT,
+      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown: haste => 90,
+      isActive: combatant => combatant.hasTalent(SPELLS.AVATAR_TALENT.id),
+      recommendedEfficiency: 0.95,
+    },
+    {
+      spell: SPELLS.STORM_BOLT_TALENT,
+      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown: haste => 30,
+      isActive: combatant => combatant.hasTalent(SPELLS.STORM_BOLT_TALENT.id),
+      recommendedEfficiency: 0.95,
+    },
+    {
+      spell: SPELLS.SHOCKWAVE_TALENT,
+      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown: haste => 40,
+      isActive: combatant => combatant.hasTalent(SPELLS.SHOCKWAVE_TALENT.id),
+      recommendedEfficiency: 0.8,
+      extraSuggestion: <span>Consider using <SpellLink id={SPELLS.DOUBLE_TIME_TALENT.id} /> or <SpellLink id={SPELLS.STORM_BOLT_TALENT.id} /> unless the CC is strictly needed.</span>,
+    },
+    {
+      spell: SPELLS.BLOODBATH_TALENT,
+      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown: haste => 30,
+      isActive: combatant => combatant.hasTalent(SPELLS.BLOODBATH_TALENT.id),
+      recommendedEfficiency: 0.95,
+    },
+    {
+      spell: SPELLS.BLADESTORM_TALENT,
+      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown: haste => 90,
+      isActive: combatant => combatant.hasTalent(SPELLS.BLADESTORM_TALENT.id),
+      recommendedEfficiency: 0.8,
+    },
+    {
+      spell: SPELLS.DRAGON_ROAR_TALENT,
+      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown: haste => 25,
+      isActive: combatant => combatant.hasTalent(SPELLS.DRAGON_ROAR_TALENT.id),
+      recommendedEfficiency: 0.8,
+    },
+    {
+      spell: SPELLS.WHIRLWIND_FURY,
+      category: Abilities.SPELL_CATEGORIES.ROTATIONAL_AOE,
+      getCooldown: haste => null,
+      noSuggestion: true,
+      noCanBeImproved: true,
+      extraSuggestion: 'Only used in a combat with AoE or as a filler with the Wrecking Ball talent',
+    },
+    {
+      spell: SPELLS.BERSERKER_RAGE,
+      category: Abilities.SPELL_CATEGORIES.UTILITY,
+      getCooldown: haste => 60,
+      noSuggestion: true,
+      noCanBeImproved: true,
+      extraSuggestion: <span>Only used in a combat with Fear, Sap or Incapacitate or if the <SpellLink id={SPELLS.OUTBURST_TALENT.id} /> is selected</span>,
+    },
+    {
+      spell: SPELLS.ENRAGED_REGENERATION,
+      category: Abilities.SPELL_CATEGORIES.DEFENSIVE,
+      getCooldown: haste => 120,
+      noSuggestion: false,
+      recommendedEfficiency: 0.01,
```

Comment:
For all of these 1% `recommendedEfficiency`s: how about either setting them to a point that encourages actually casting them (e.g. 60%) or just removing the prop and adding:
```
      noSuggestion: true,
      noCanBeImproved: true,
```
?